src/abstractdatatable/abstractdatatable.jl

```diff
@@ -438,11 +438,13 @@ Indexes of complete cases (rows without null values)
 
 ```julia
 completecases(dt::AbstractDataTable)
+completecases(dt::AbstractDataTable, cols)
 ```
 
 **Arguments**
 
 * `dt` : the AbstractDataTable
+* `cols` : a column indicator (Symbol, Int, Vector{Symbol}, etc.) specifying the column(s) to check for Nulls
```

Comment:
That's actually an issue in the existing docstrings for similar functions. An additional commits to fix these would be welcome though.